tests/baselines/reference/inferPartialTypeArguments.errors.txt

```diff
@@ -0,0 +1,103 @@
+tests/cases/conformance/types/typeParameters/typeParameterLists/inferPartialTypeArguments.tsx(23,10): error TS7026: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists.

```

Comment:
Hm, I swear I had this whole test file working - perhaps some last-minute changes broke that. I'll investigate this.